tools/pom.xml

```diff
@@ -49,6 +49,13 @@
             <version>${log4j.version}</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Ah. I stumbled over this because dropwizard has its "dropwizard-testing", which includes a junit version.

Why do we have dependencies in the parent pom? Do they belong there? Parents often do "dependency management", where they say to children "**if** you need dependency A, then use version a.b". But ultimately only the modules by themselves can know what dependencies they have.